twoliter/src/cmd/build.rs

```diff
@@ -114,6 +114,10 @@ pub(crate) struct BuildVariant {
     /// from the upstream URL found in a package's `Cargo.toml`.
     #[clap(long = "upstream-source-fallback")]
     upstream_source_fallback: bool,
+
+    /// Path to the Infra.toml file
+    #[clap(long)]
+    infra_toml: Option<PathBuf>,
```

Comment:
This will also be needed for `twoliter publish variant` and `twoliter publish kit` - should it be some sort of common argument or is it easier just to add it to each command?

Reply:
There will probably be some refactoring opportunities to make common arguments. Since publish variant and publish kit aren't there yet, I just went with what was immediately requested.